src/test/java/uk/sky/cqlmigrate/LockVerificationTest.java

```diff
@@ -166,31 +185,17 @@ public void shouldObtainLockToInsertRecord() throws InterruptedException {
     }
 
     private void updateCounter(int updatedCounter) {
-        Statement updateCounterQuery = new SimpleStatement(String.format("UPDATE %s SET counter = %d WHERE id = 'lock-tester'", TABLE_NAME, updatedCounter)).setConsistencyLevel(ConsistencyLevel.QUORUM);
+        Statement updateCounterQuery = SimpleStatement.newInstance(String.format("UPDATE %s SET counter = %d WHERE id = 'lock-tester'", TABLE_NAME, updatedCounter)).setConsistencyLevel(ConsistencyLevel.QUORUM);
         session.execute(updateCounterQuery);
 
     }
 
     private int readCurrentCounter() {
-        Statement readCounterQuery = new SimpleStatement(String.format("SELECT counter from %s WHERE id = 'lock-tester'", TABLE_NAME)).setConsistencyLevel(ConsistencyLevel.QUORUM);
+        Statement readCounterQuery = SimpleStatement.newInstance(String.format("SELECT counter from %s WHERE id = 'lock-tester'", TABLE_NAME)).setConsistencyLevel(ConsistencyLevel.QUORUM);
         final ResultSet currentRecord = session.execute(readCounterQuery);
         return currentRecord.one().getInt("counter");
     }
 
-    private Cluster createCluster() {
-        QueryOptions queryOptions = new QueryOptions();
-        queryOptions.setConsistencyLevel(ConsistencyLevel.LOCAL_QUORUM);
-
-        SocketOptions socketOptions = new SocketOptions();
-        socketOptions.setReadTimeoutMillis(1000);
-
-        return Cluster.builder()
-                .addContactPoints(CASSANDRA_HOST)
-                .withPort(CASSANDRA_PORT)
-                .withQueryOptions(queryOptions)
-                .withSocketOptions(socketOptions)
-                .build();
-    }
```

Comment:
We no longer specify timeouts  in this test. Were they needed?

Reply:
Since we no longer have to create a cluster with the new driver, the closest option for session I found was REQUEST_TIMEOUT and have added that to the test. This test is ignored and doesnt pass when we run locally either so makes it harder to know if this works or not. Happy to create an issue to look at it for both this driver and the old driver.

Reply:
Create an issue please (to fix and enable)! Otherwise happy